packages/kernel-5.10/config-bottlerocket

```diff
@@ -103,6 +103,13 @@ CONFIG_MOUSE_PS2=m
 
 # Add support for IPMI drivers
 CONFIG_IPMI_HANDLER=m
+# Disable more specialized IPMI drivers that are not relevant for our use-cases
+# CONFIG_IPMI_DEVICE_INTERFACE is not set
+# CONFIG_IPMI_PANIC_EVENT is not set
+# CONFIG_IPMI_POWEROFF is not set
+# CONFIG_IPMI_SI is not set
+# CONFIG_IPMI_WATCHDOG is not set
+# CONFIG_ACPI_IPMI is not set
```

Comment:
That configuration initially confused me. Nothing is exposed to the user space (e.g. no support for `ipmitool`) and there's no system interface drivers (`CONFIG_IPMI_SI`, `CONFIG_IPMI_SSIF` are deselected) to talk to a BMC, so the IPMI message handler would seem to be quite useless. However, `CONFIG_IPMI_HANDLER=m` was set in #1862 to support NVIDIA variants, so I assume NVIDIA drivers provide a bespoke driver. Would you mind adding that to the comment about adding support for IPMI drivers in case you send another revision? It's fine if not, I can do it later.